### DISCUSSION TITLE
CitraVR (Beta) v0.2

### DISCUSSION THREAD

**Discussion by amwatson** [amwatson/CitraVR#3] [Announcements]
- Fixed an edge-case where receiving notifications can trigger the 2D activity to switch orientations 

<hr /><em>This discussion was created from the release <a href='https://github.com/amwatson/CitraVR/releases/tag/v0.2.0'>CitraVR (Beta) v0.2</a>.</em>

**Comment by artsin1981** [comment]
Impressive project! However, it appears that fewer games are functional when compared to the Android and PC Citra Emulators. Specifically, Eshop decrypted titles like Streets of Rage, Gunstar Heroes, and Fantasy Zone 1/2 seem to be affected. Additionally, titles like Ridge Racer, which work seamlessly on other versions, face issues here. 

Would be nice to have an option to exit back to roms menu too.

**Comment by bsharper** [comment]
Just wanted to say that I love this project so much. I wasn't expecting things to be playable so early on, but I just looked at my watch and realized I'd already been playing for a few hours :) Thank you!
